reana_job_controller/job_monitor.py

```diff
@@ -237,3 +293,10 @@ def format_condor_job_que_query(backend_job_ids):
     for job_id in backend_job_ids:
         query += base_query.format(job_id)
     return query[:-2]
+
+
+def format_slurm_job_query(backend_job_ids):
```

Comment:
Since this belongs only to Slurm, can you move it inside a superclass? i.e. `JobMonitorSlurm`